OSDOCS-2191: New metrics for LSO (#36567)
openshift/openshift-docs: modules/persistent-storage-local-metrics.adoc

// Module included in the following assemblies:
//
// * storage/persistent_storage/persistent-storage-local.adoc

[id="local-storage-metrics_{context}"]
= Local Storage Operator Metrics

{product-title} provides the following metrics for the Local Storage Operator:

> * lso_discovery_disk_count: total number of discovered devices on each node

Reviewer: ```suggestion
* `lso_discovery_disk_count`: total number of discovered devices on each node
```

This is the snippet from our doc guidelines. This comment applies to all of the bullets in the list: 
> Use backticks for all non-GUI "system items", including: Inline commands, operations, literal values, variables, parameters, settings, flags, environment variables, user input. System term/item, user names, unique or example names for individual API objects/resources (e.g., a pod named mypod), daemon, service, or software package. RPM packages. File names or directory paths

* lso_lvset_provisioned_PV_count: total number of PVs created by `LocalVolumeSet` objects

* lso_lvset_unmatched_disk_count: total number of disks that Local Storage Operator did not select for provisioning because of mismatching criteria

* lso_lvset_orphaned_symlink_count: number of devices with PVs that no longer match `LocalVolumeSet` object criteria

* lso_lv_orphaned_symlink_count: number of devices with PVs that no longer match `LocalVolume` object criteria

> * lso_lv_provisioned_PV_count: total number of provisioned PVs for L`ocalVolume`

Reviewer: ```suggestion
* lso_lv_provisioned_PV_count: total number of provisioned PVs for `LocalVolume`
```

To use these metrics, be sure to:

* Enable support for monitoring when installing the Local Storage Operator.

> * When upgrading to {product-title} 4.9 or later, enable metric support manually by adding the operator-metering=true label to the namespace.

Reviewer: ```suggestion
* When upgrading to {product-title} 4.9 or later, enable metric support manually by adding the `operator-metering=true` label to the namespace.
```

This is the snippet from our doc guidelines. 

> Use backticks for all non-GUI "system items", including: Inline commands, operations, literal values, variables, parameters, settings, flags, environment variables, user input. System term/item, user names, unique or example names for individual API objects/resources (e.g., a pod named mypod), daemon, service, or software package. RPM packages. File names or directory paths
